content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -27,7 +37,7 @@ binaryTargets = ["native", "rhel-openssl-1.0.x"]
 
 This is necessary because the runtime used in the development and when your application is deployed differ. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine file available.
 
-### Lambda functions with arm64 architectures
+#### Lambda functions with arm64 architectures
 
 Lambda functions that use [arm64 architectures (AWS Graviton2 processor)](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html#foundation-arch-adv) must use an `arm64` precompiled engine file.
```

Comment:
Is Graviton still the only relevant one?

Reply:
Yes, according to the linked docs there are "x86-based processors" and "AWS Graviton2 processor".